docs/theme/components/Menu.styles.ts

```diff
@@ -1,10 +1,33 @@
 import { ThemeComponent, cva } from '@marigold/system';
 
 export const Menu: ThemeComponent<'Menu'> = {
```

Comment:
Before on mobile, it took the whole width. It's a bit small now

Reply:
As I read this, it was always this applied: `size-full w-[500px] ` which doesn't make it look good on mobile: 
![Bildschirmfoto 2024-04-26 um 08 22 07](https://github.com/marigold-ui/marigold/assets/38324334/4b80c550-a555-4840-8e7b-2b9dcb571456)